src/App.tsx

```diff
@@ -1,24 +1,80 @@
-import React, { FC } from 'react';
-import logo from './logo.svg';
+import React, { FC, useState } from 'react';
 import './App.css';
+import { Button, TextField } from '@material-ui/core';
+import Task from './components/Task';
+import { TaskInterface } from './utils/Interfaces';
+import useStyles from './styles/styles';
 
-const App: FC = () => (
-  <div className="App">
-    <header className="App-header">
-      <img src={logo} className="App-logo" alt="logo" />
-      <p>
-        Edit <code>src/App.tsx</code> and save to reload.
-      </p>
-      <a
-        className="App-link"
-        href="https://reactjs.org"
-        target="_blank"
-        rel="noopener noreferrer"
-      >
-        Learn React
-      </a>
-    </header>
-  </div>
-);
+const App: FC = () => {
+  const [todoList, setTodoList] = useState<TaskInterface[]>([]);
+  const [input, setInput] = useState<string>('');
+  const [error, setError] = useState<string>('');
+  const styles = useStyles();
+
+  const addTask = () => {
+    if (!input || input.length < 6 || input.length > 20) {
+      setError('Entrada inválida (digite de 6 a 20 caracteres)');
+    } else {
+      setTodoList([{
+        title: input,
+        done: false,
+      }, ...todoList])
+      setError('');
+    }
+  };
+
+  const changeStatus = (position: number) => {
+    setTodoList(
+      todoList.map((task, index) => index === position
+        ? Object.assign(task, { done: !task.done })
+        : task)
+    )
+  }
+
+  const removeTask = (position: number) => {
```

Comment:
I suggest changing name to `removeTodoHandler`.

Reply:
Resolved on 5d62c7f44645c0fdd2d223a0a7a151e9f2aaf553